src/test/java/core/basesyntax/service/operations/BalanceOperationHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.service.operations;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.model.Operation;
+import core.basesyntax.service.OperationHandler;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+public class BalanceOperationHandlerTest {
+
+    private static OperationHandler balanceOperation;
+
+    @Before
+    public void before() {
+        balanceOperation = new BalanceOperationHandler();
+    }
+
+    @Test(expected = NullPointerException.class)
```

Comment:
you should not expect catching NPE - fix in other tests as well

Reply:
done